nixos/modules/services/web-servers/nginx/vhost-options.nix

```
@@ -238,6 +238,8 @@ with lib;
         Note that HTTP/3 support is experimental and
         *not* yet recommended for production.
         Read more at https://quic.nginx.org/
+        It is also recommended manually add advertise the HTTP/3
+        availability to the location blocks.
```

```suggestion
        Note that HTTP/3 support is experimental and *not* yet recommended for production.
        Read more at <https://quic.nginx.org/>.
        It is also recommended manually add advertise the HTTP/3 availability to the location blocks.
```
We don't need to limit ourselves to those very short line lengths to reduce readability. The other lines have long links though.